README.md

```diff
@@ -15,17 +15,18 @@ LinuxKit, a toolkit for building custom minimal, immutable Linux distributions.
 
 ## Getting Started
 
-### Build the `moby` tool
+### Build the `moby` and `linuxkit` tools
```

Comment:
yes. And the `moby` tool will be moved and extended to build assemblies other than linux images. This was discussed at length at DockerCon and various meetings after. I'm not sure there is a good writeup yet